tests/baselines/reference/doYouNeedToChangeYourTargetLibraryES2016Plus.errors.txt

```diff
@@ -86,13 +87,13 @@ tests/cases/compiler/doYouNeedToChangeYourTargetLibraryES2016Plus.ts(44,33): err
 !!! error TS2583: Cannot find name 'AsyncGenerator'. Do you need to change your target library? Try changing the 'lib' compiler option to 'es2018' or later.
     const testAsyncGeneratorFunction: AsyncGeneratorFunction = null as any;
                                       ~~~~~~~~~~~~~~~~~~~~~~
-!!! error TS2583: Cannot find name 'AsyncGeneratorFunction'. Do you need to change your target library? Try changing the 'lib' compiler option to 'es2018' or later.
+!!! error TS2552: Cannot find name 'AsyncGeneratorFunction'. Did you mean 'GeneratorFunction'?
     const testAsyncIterable: AsyncIterable<any> = null as any;
                              ~~~~~~~~~~~~~
 !!! error TS2583: Cannot find name 'AsyncIterable'. Do you need to change your target library? Try changing the 'lib' compiler option to 'es2018' or later.
     const testAsyncIterableIterator: AsyncIterableIterator<any> = null as any;
                                      ~~~~~~~~~~~~~~~~~~~~~
-!!! error TS2583: Cannot find name 'AsyncIterableIterator'. Do you need to change your target library? Try changing the 'lib' compiler option to 'es2018' or later.
+!!! error TS2552: Cannot find name 'AsyncIterableIterator'. Did you mean 'IterableIterator'?

```

Comment:
This worries me a bit.

Reply:
Is there an easy way to do the built-in declarations check before the "did you mean...?" check?

Reply:
These are restored with the latest commit. Even added some more lib recommendations related to BigInt64Array.